From 71b0c4625ab94d7632a061dccaf665fce80dd22c Mon Sep 17 00:00:00 2001
From: Alethea Rose <alethea@alethearose.com>
Date: Tue, 14 Feb 2017 22:26:28 -0500
Subject: [PATCH 03/23] Include win32-headers.h in v8/atomicops.h

[jes: this may not be needed anymore... another instance where a good
commit message would save a ton of time!]
---
 deps/v8/src/base/atomicops.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/deps/v8/src/base/atomicops.h b/deps/v8/src/base/atomicops.h
index c4c28f70..30384303 100644
--- a/deps/v8/src/base/atomicops.h
+++ b/deps/v8/src/base/atomicops.h
@@ -36,6 +36,10 @@
 #include "src/base/base-export.h"
 #include "src/base/build_config.h"
 
+#if defined(_WIN32)
+#include "win32-headers.h"
+#endif
+
 namespace v8 {
 namespace base {
 
-- 
2.17.0.windows.1

